griptape/events/action_chunk_event.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional
+
+from attrs import define, field
+
+from griptape.events.base_chunk_event import BaseChunkEvent
+
+if TYPE_CHECKING:
+    from griptape.common import BaseDeltaMessageContent
+
+
+@define
+class ActionChunkEvent(BaseChunkEvent):
+    tag: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    name: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    path: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+
+    @classmethod
+    def from_delta_message_content(cls, content: BaseDeltaMessageContent) -> ActionChunkEvent:
+        from griptape.common import ActionCallDeltaMessageContent
+
+        if isinstance(content, ActionCallDeltaMessageContent):
+            return cls(
+                token=content.partial_input if content.partial_input is not None else "",
+                index=content.index,
+                tag=content.tag,
+                name=content.name,
+                path=content.path,
+            )
+
+        raise ValueError(f"Content is not an instance of ActionCallDeltaMessageContent: {content.__class__.__name__}")
```

Comment:
We use this pattern on Message Contents because we're aggregating many Delta Message Contents into a full Message Content. We don't need this on Events.